rubrics/impl/src/main/java/org/sakaiproject/rubrics/impl/RubricsServiceImpl.java

```diff
@@ -876,11 +870,19 @@ public Optional<ToolItemRubricAssociation> saveRubricAssociation(String toolId,
                             }
                         }
                     }
+                } else if (requestedRubricId.equals(0L)){   //deactivating an association without making a new one
+                    association.setActive(false);
+                    associationRepository.save(association);
+                    return Optional.empty();
+                }
+            } else {    //if existingAssociation is not present, it could just mean that it was deactivated previously; the specific getRubricAssociation impl that we used earlier to load it will ignore deactivated ones.
```

Comment:
Could you put this comment inside the block, rather than after the brace? It's a big comment and makes things harder to read, imo. Also, these comments probably should go on the api method, if they help us understand the code.

Reply:
I'll put this one inside the block and then add some separate info on the API method if that's ok.